public/index.html

```diff
@@ -25,6 +25,7 @@
     <link href="https://fonts.gstatic.com" rel="preconnect">
     <link href="https://fonts.googleapis.com/css2?family=Blinker:wght@100;200;300;400;600;700;800;900&display=swap"
           rel="stylesheet">
+          <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
Would be nice to serve fonts and the CSS from next and not an external CDN.

We should switch out the fonts to whatever we're currently using on the main site... we were using `Electrolize`, but also experimented with `Jost`. https://github.com/public-awesome/stargaze-website/blob/main/styles/sass/util/fonts.scss